### PR TITLE
Add upgrade checklist template

### LINKED ISSUE #1354: Create Upgrade Checklist Template
## Summary

Wanted to start taking notes about what it should include here:

* testnet
  * communication prep & docs update
  * RC
  * run for a week
  * final release
* docs - prepare `update-docs` branch on these repos before upgrade:
  * https://github.com/cosmos/testnets
  * https://github.com/cosmos/mainnet
  * https://github.com/cosmos/gaia
    * joining mainnet
    * quickstart
    * README.md
    * changelog
      * Add CLI / REST api changes
* proposal
  * generate binaries in release
  * predict block height for target date
* communication
  * blogposts
    * testnet - target validators on Interchain Medium
    * mainnet - target wider audience on Cosmos Medium
  * twitter 
    * link to testnet upgrade blog - @ cosmohub
    * link to mainnet upgrade blog -  @ cosmos @ cosmoshub
    * upgrade countdown -  @ cosmos @ cosmoshub
    * upgrade success story -  @ cosmos @ cosmoshub
  * discord, telegram, slack
    * testnet upgrade info (discord only)
    * link to mainnet upgrade instructions (all channels)
  * validator email list
    * upgrade instructions + link to mainnet blog - one week before upgrade
* update [chain registry](https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json)

(this isn't the most sensical format but just wanted a place to start brain dumping the things we're currently doing so we don't leave them out of the eventual template)

## Proposal

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

### PR DESCRIPTION
**Pull request by nooomski**
Closes #1354

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/upgrade-checklist.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Cosmos Hub Upgrade Epic
+
+# <Upgrade Name>
+
+Check mark each item if it has been completed and ready for the upgrade. If not added yet, create an issue for each item and mark complete once it has been done and integrated.
+
+- [ ]  Upgrade to SDK version <SDK VERSION>
+- [ ]  Upgrade to IBC version <IBC VERSION>
+- [ ]  Integrate new modules
```
Comment (glnro) [inline .github/ISSUE_TEMPLATE/upgrade-checklist.md]: Maybe link to https://github.com/cosmos/hub-eng/blob/main/module_qa/module_checklist.md or maybe we combine that with the existing module checklist template

**Review by glnro — APPROVED** [review]
LGTM, maybe adding a space under each major deliverable to include the owner

File: .github/ISSUE_TEMPLATE/upgrade-checklist.md (hunk 1 above)
Comment (nooomski) [reply .github/ISSUE_TEMPLATE/upgrade-checklist.md]: Cool, added a link. Wdym with adding a space to include an owner?

Comment (nooomski) [reply .github/ISSUE_TEMPLATE/upgrade-checklist.md]: @glnro lemme know, otherwise happy to merge if you are.

Comment (glnro) [reply .github/ISSUE_TEMPLATE/upgrade-checklist.md]: lgtm

**Pull request merged**